Tests/KnitCodeGenTests/KnitDirectivesTests.swift

```diff
@@ -128,6 +128,13 @@ final class KnitDirectivesTests: XCTestCase {
         )
     }
 
+    func testCustom() {
+        XCTAssertEqual(
+            try parse("// @knit custom(\"Foo\") custom(\"Bar\")"),
```

Comment:
```suggestion
            try parse("// @knit tag(\"Foo\") tag(\"Bar\")"),
```
Naming suggestion: "tag"

Reply:
That's better than custom. I've updated the PR to use tag